README.md

```diff
@@ -6,9 +6,16 @@
 
 ![logo](https://raw.githubusercontent.com/opentok/accelerator-core/master/tokbox-logo.png)
 
-# OpenTok Accelerator Core
+# OpenTok Accelerator Core Web
 
+OpenTok's `Accelerator Core Web` provides a simple and effortless integration for OpenTok libraries. Wether you already built your application or are starting to use OpenTok's libraries, `Accelerator Core Web` gives you an easy and fast way to integrate audio/video communication into your application.
```

Comment:
See comments below on "The regular Initialization for `OpenTok Accelerator Core` is".  We may want something like that in the summary at the top since what is currently there seems to be repeated in `Quickstart`.